src/main/webapp/WEB-INF/templates/user_dashboard.jsp

```diff
@@ -19,21 +19,38 @@
   </div>
 </head>
 <body>
-    <form class="setting" action="/user" method="POST">
-      <div class="row">
-        <label for="input-websites">Choose your preferred website:</label>
-        <select class="input-websites" id="input-websites" name="website">
-          <option value="GeeksForGeeks">GeeksForGeeks</option>
-          <option value="StackOverflow">StackOverflow</option>
-          <option value="W3Schools">W3Schools</option>
-        </select>
-      </div>
-      <div class="row">
-        <label for="input-languages">Choose your primary coding language:</label>
-        <input class="input-languages" id="input-languages" type="text" name="language"
-               placeholder="Input your primary coding language">
-      </div>
-      <input class="submit-button" type="submit" value="Submit">
-    </form>
+<a class="home-button" href="/">Home</a>
+<form class="setting" action="/user" method="POST">
+  <div class="row">
+    <label for="input-websites">Choose your preferred website:</label>
+    <select class="input-websites" id="input-websites" name="website">
+      <c:choose>
+        <c:when test="${user.getWebsite() != null}">
+          <option value="${user.getWebsite()}" selected hidden>${user.getWebsite()}</option>
+        </c:when>
+        <c:otherwise>
+          <option value="" selected disabled hidden>Select</option>
+        </c:otherwise>
+      </c:choose>
+      <option value="GeeksForGeeks">GeeksForGeeks</option>
+      <option value="StackOverflow">StackOverflow</option>
+      <option value="W3Schools">W3Schools</option>
+    </select>
+  </div>
+  <div class="row">
+    <label for="input-languages">Choose your primary coding language:</label>
+    <input class="input-languages" id="input-languages" type="text" name="language"
+    <c:choose>
+    <c:when test="${user.getLanguage() != null}">
```

Comment:
Also check for empty string (and above). If I submit the form without editing any fields, by default the language will be sent as empty string.